pkg/generate/code/set_resource.go

```diff
@@ -258,6 +258,8 @@ func SetResource(
 		out += fmt.Sprintf(
 			"%sif %s != nil {\n", indent, sourceAdaptedVarName,
 		)
+		qualifiedTargetVar := fmt.Sprintf("%s.%s",
```

Comment:
femto-nit: Generally if I need to wrap a function call at 80 chars, I use the style like lines 258-260 above, where I have opening parentheses, newline, args-wrapped-at-80-chars, newline, and closing parens on separate line. So, this would be:

```go
		qualifiedTargetVar := fmt.Sprintf(
			"%s.%s", targetAdaptedVarName, f.Names.Camel,
		)
```